scripts/config.ini

```diff
@@ -11,30 +11,118 @@ ssh = ssh {CFN_USER}@{MASTER_IP} {ARGS}
 
 [cluster hpc]
 key_name = ${ssh_key_id}
-base_os = ubuntu1804
+#base_os = ubuntu1804
+base_os = ${base_os_choice}
 scheduler = slurm
 master_instance_type = c5.2xlarge
-compute_instance_type = c5.18xlarge
 vpc_settings = public-private
-fsx_settings = fsx-scratch2
-disable_hyperthreading = true
+fsx_settings = fsx-mount
+queue_settings = c5-4xlarge, c5-4xlarge-spot, c5n-18xlarge, g4dn-12xlarge
 dcv_settings = dcv
+scaling_settings = custom
 post_install = ${post_install_script_url}
-post_install_args = "/shared/spack-0.15 releases/v0.15 /opt/slurm/log sacct.log"
+post_install_args = "/shared/spack-0.16 releases/v0.16 /opt/slurm/log sacct.log"
 s3_read_resource = arn:aws:s3:::*
 s3_read_write_resource = ${s3_read_write_resource}/*
-initial_queue_size = 0
-max_queue_size = 10
-placement_group = DYNAMIC
 master_root_volume_size = 50
-compute_root_volume_size = 30
 ebs_settings = myebs
 cw_log_settings = cw-logs
 
+[scaling custom]
+scaledown_idletime = 10
+
+[queue c5-4xlarge]
+compute_resource_settings = c5-4xlarge
+compute_type = ondemand
+enable_efa = false
+enable_efa_gdr = false
+disable_hyperthreading = true
+
+[compute_resource c5-4xlarge]
+instance_type = c5.4xlarge
+min_count = 0
+max_count = 16
+initial_count = 0
+
+
+[queue c5-4xlarge-spot]
+compute_resource_settings = c5-4xlarge-spot
+compute_type = spot
+enable_efa = false
+enable_efa_gdr = false
+disable_hyperthreading = true
+
+[compute_resource c5-4xlarge-spot]
+instance_type = c5.4xlarge
+min_count = 0
+max_count = 16
+# If you don't specify a value, you're charged the Spot price, capped at the
```

Comment:
remove

Reply:
I wanted the based config to show multi-queue example with on-demand, spot, efa, and gpu queues. (i.e., queue_settings = c5-4xlarge, c5-4xlarge-spot, c5n-18xlarge, g4dn-12xlarge). Removed the excess queues for p3dn-24xlarge and p4d-24xlarge.